test/Sign.Cli.Test/AzureCredentialOptionsTests.cs

```diff
@@ -191,7 +191,6 @@ public void CreateDefaultAzureCredentialOptions_WhenNoOptionsAreSpecified_Exclud
             DefaultAzureCredentialOptions credentialOptions = _options.CreateDefaultAzureCredentialOptions(result);
 
             Assert.True(credentialOptions.ExcludeInteractiveBrowserCredential);
-            Assert.True(credentialOptions.ExcludeSharedTokenCacheCredential);
```

Comment:
This fixes a build error:

```
error CS0618: 'DefaultAzureCredentialOptions.ExcludeSharedTokenCacheCredential' is obsolete: 'SharedTokenCacheCredential is deprecated. Consider using other dev tool credentials, such as VisualStudioCredential.'
```

https://github.com/Azure/azure-sdk-for-net/pull/51587